src/components/datePicker/DatePickerDropdown.tsx

```diff
@@ -157,9 +157,13 @@ export class DatePickerDropdown extends React.Component<IDatePickerDropdownProps
         </footer>
       )
     };
-    let datePickerBox: JSX.Element = this.props.withReduxState
-      ? <DatePickerBox withReduxState id={this.props.id} {...datePickerBoxProps} />
-      : <DatePickerBox {...datePickerBoxProps} />;
+
+    let datePickerBox: JSX.Element = null;
```

Comment:
Thanks, I did, the only place it is used is in the JSX element that is returned by the method, we use null in JSX when we don't want any dom elements to be rendered on the page. Tried it in my use case as well, and others (e.g. log browser) and nothing breaks, it's just way faster to render.